tests/evm/test_returndatasize.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+
+from zkevm_specs.evm import (
+    ExecutionState,
+    StepState,
+    verify_steps,
+    Tables,
+    CallContextFieldTag,
+    Bytecode,
+    RWDictionary,
+)
+from zkevm_specs.util import rand_fq, RLC, U64
+
+
+TESTING_DATA = (
+    0x00,
+    0x10,
+    0x302010,
+)
+
+
+@pytest.mark.parametrize("returndatasize", TESTING_DATA)
+def test_returndatasize(returndatasize: U64):
+    randomness = rand_fq()
+
+    bytecode = Bytecode().returndatasize()
+    bytecode_hash = RLC(bytecode.hash(), randomness)
+
+    tables = Tables(
+        block_table=set(),
+        tx_table=set(),
+        bytecode_table=set(bytecode.table_assignments(randomness)),
+        rw_table=set(
+            RWDictionary(9)
+            .call_context_read(1, CallContextFieldTag.CallDataLength, returndatasize)
```

Comment:
`CallContextFieldTag.CallDataLength` should be `CallContextFieldTag.LastCalleeReturnDataLength`, right?

Reply:
Agree!